index.html

```diff
@@ -0,0 +1,92 @@
+      </a>
+    </div>
+
+    <div class="Product-tile">
+      <a href="https://product-landing-page.freecodecamp.rocks/" target="_blank"> Product Page
+      <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6d/Posaune.jpg/250px-Posaune.jpg" alt="">  
+      </a>
+    </div>
+
+    <div class="Technical-tile">
+      <a href="https://technical-documentation-page.freecodecamp.rocks/" target="_blank"> Documentation Page 
+     <img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Ftechbeacon.com%2Fapp-dev-testing%2Fwhy-agile-teams-should-care-about-documentation&psig=AOvVaw2cBbX2UNpITL7c4izuV9R0&ust=1667565688991000&source=images&cd=vfe&ved=0CA0QjRxqFwoTCPjJ_LWEkvsCFQAAAAAdAAAAABAI" alt="">
+
+      </a>
+    </div>
+  
+  </div>
+
+
+
+    </section>
+
+    <footer>
+      <h2>GET IN TOUCH HERE</h2>
+    <section id="profiles">
+      <div class="profile-links">
+        <a id="profile-link" href="https://github.com/frempongdev" target="_blank" > GitHub</a>
+        <a href="https://twitter.com/" target="_blank" > Twitter</a>
+        <a href="mailto:samankwaa.frempong@gmsil.com"> Mail</a>
+      </div>
```

Comment:
Great implementation of `<a>` tag your footer 🔥 . However, because there are three of them, it is better semantically to make them list items. You can implement a `<ul>` with three `<li>` items for the individual links.

Reply:
Good job 👍